vips/vip-502/bsctestnet.ts

```diff
@@ -0,0 +1,130 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+import { PSR } from "../../multisig/proposals/basesepolia/vip-003";
+import { PLP, PRIME } from "../../multisig/proposals/basesepolia/vip-007";
+
+export const BOUND_VALIDATOR = "0xC76284488E57554A457A75a8b166fB2ADAB430dB";
+export const POOL_REGISTRY = "0xCa330282BEeb07a81963336d0bf8f5f34317916c";
+export const VTOKENS = [
+  "0x776f14D624aBdAfa912d6Cd0864976DdaF5Ca4a7",
+  "0x436E5A07F58AAA86277e8b992bC3e596eC423d09",
+  "0xA31D67c056Aadc2501535f2776bF1157904f810e",
+];
+export const COMPTROLLERS = ["0x272795dd6c5355CF25765F36043F34014454Eb5b"];
+export const XVS_STORE = "0x059f1eA3973738C649d63bF4dA18221ecA418cDC";
+export const XVS_BRIDGE_ADMIN = "0xE431E82d8fFfd81E7c082BeC7Fe2C306f5c988aD";
+export const XVS_BRIDGE = "0xD5Cd1fD17B724a391C1bce55Eb9d88E3205eED60";
```

Comment:
Why don't we import every address from `"../../multisig/proposals/basesepolia/vip-007"`? I think it would simplify the review process and reduce the chances to make an error